test/trento/domain/cluster/cluster_test.exs

```diff
@@ -777,6 +778,120 @@ defmodule Trento.ClusterTest do
   end
 
   describe "deregistration" do
+    test "should restore a deregistered cluster when a RegisterHost command from a non DC host is received" do
```

Comment:
Same here, don't we mean `RegisterClusterHost`?

Reply:
thx!